controls/ssh_spec.rb

```diff
@@ -22,6 +22,8 @@
   command('ssh').exist?
 end
 
+ssh_version = command('ssh -V 2>&1 | cut -f1 -d" " | cut -f2 -d"_"').stdout.to_f
```

Comment:
Should we do this in a resource, so that we could reuse it for other test too in the future?